src/components/SignUp/SignUp.css

```diff
@@ -93,4 +95,11 @@
   50% {
     transform: scale(80%);
   }
-}
+
```

Comment:
![image](https://user-images.githubusercontent.com/59261812/123796652-fa73ff00-d902-11eb-968c-8cd6cd477cd2.png)

Reply:
This might happen because of conflict file because this line is not available in my code